docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -114,6 +129,13 @@ If your module does not need to target a remote host, you can quickly and easily
 
    $ python library/my_test.py /tmp/args.json
 
+- For a module developed in an existing collection called ``my_namespace.my_collection``, you may need to do:
```

Comment:
```suggestion
You might also need to add your collections to the Python path, as in the following example:
```

Reply:
I feel like this intro sentence should address the detail about the Python path environment variable. @Akasurde when is it necessary to use this? Is it always required or are there certain conditions that require using the PYTHONPATH variable?

Reply:
It is always required in the case of module development in an existing collection.